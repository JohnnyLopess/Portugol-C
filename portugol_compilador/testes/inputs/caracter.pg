programa
{
    funcao inicio() {
        caracter letra
        letra = 65
        escreva(letra)
    }
}
