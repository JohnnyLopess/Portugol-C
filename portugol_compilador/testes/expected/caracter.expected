#include <stdio.h>
#include <string.h>

int main() {
    char letra;
    letra = 65;
    printf("%c\n", letra);
    return 0;
}
